### ISSUE TITLE
Top right tooltip disappears when activating the verifier

### ISSUE THREAD

**Issue by revolter** [maruohon/litematica#47]
Before schematic verifier | After
--- | ---
![Screenshot 2019-07-23 at 00 45 34](https://user-images.githubusercontent.com/5748627/61667659-45cb0f00-ace3-11e9-9708-22ea3e9087cb.png) | ![Screenshot 2019-07-23 at 00 45 47](https://user-images.githubusercontent.com/5748627/61667667-4cf21d00-ace3-11e9-8158-3db3fea6b20f.png)

**Comment by maruohon** [comment]
Yeah this issue is already on my TODO list to fix. It also breaks the Easy Place mode when the verifier markers exist in the target position. I basically need to add a flag to not consider the verifier markers in the ray traces that are used for the block info line, and the Easy Place mode, and probably also some other places.
